Configuration/Eras/python/Era_Run2_HI_2018_cff.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Eras.Modifier_run2_common_cff import run2_common
+from Configuration.Eras.Modifier_run2_HI_specific_cff import run2_HI_specific
+from Configuration.Eras.Modifier_stage1L1Trigger_cff import stage1L1Trigger
+from Configuration.Eras.Modifier_run2_HI_2018_cff import run2_HI_2018
+ 
+Run2_HI_2018 = cms.ModifierChain(run2_common, run2_HI_specific, stage1L1Trigger, run2_HI_2018)
```

Comment:
at a closer look, this will not work for 2018.
This is really just Run2_HI_emulatedZS10 (or what was the name originally introduced by @CesarBernardes ).
Sorry for suggesting a different name, I thought that this was the production version

Reply:
yes, it can be removed.
We will still probably need a Run2_HI_2018 for actual 2018 data processing.
The modifier has to be known in Configuration/StandardSequences/python/Eras.py to be passable to the cmsDriver.

Reply:
OK thanks.

Reply:
so, we still need to get rid of this file/era [just repeating for the last round of review]